### ISSUE TITLE
Import is used?

### ISSUE THREAD

**Issue by ZergsLaw** [yoheimuta/go-protoparser#87]
How I can get information - import is used or not?

**Comment by yoheimuta** [comment]
@ZergsLaw Yes. Import is defined in https://pkg.go.dev/github.com/yoheimuta/go-protoparser/v4@v4.10.0/parser#Import.
You can see the example output here:https://github.com/yoheimuta/go-protoparser/blob/master/README.md?plain=1#L129-L148.

**Comment by ZergsLaw** [comment]
I mean, is there a way to check that a specific import is being used?

For example - myVarWithImport.IsUsed() ?

**Comment by 12345dotblue** [comment]
> @ZergsLaw Yes. Import is defined in https://pkg.go.dev/github.com/yoheimuta/go-protoparser/v4@v4.10.0/parser#Import. You can see the example output here:https://github.com/yoheimuta/go-protoparser/blob/master/README.md?plain=1#L129-L148.

How to set the include path which contains the proto files ?
